pmm-app/src/shared/components/Form/InputLabel/InputLabel.style.ts

```diff
@@ -0,0 +1,18 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  inputLabel: css`
+      align-items: center;
+      background-color: ${theme.colors.bg3};
+      border-radius: 3px 0 0 3px;
```

Comment:
@nicolalamacchia Isn't there a borderRadius that can be taken from the theme?

Reply:
You're right, there is! Updated